### DISCUSSION TITLE
Unclear instructions on requesting purchase approval

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/console#63] [Craft Console] [answered]
I wasn't sure if our client had added a payment method yet, so I tried to purchase a CMS license and plugin license. The message I received indicates that they haven't added a payment method, but I can still request approval; however, it's unclear how I should proceed with requesting approval or how that process will work.

![image](https://user-images.githubusercontent.com/3810939/203587981-e598f29a-7b2a-4c05-9dae-6654cbd4b4bc.png)

**Comment by darylknight** [comment]
Also just run into this. Brandon's just confirmed they're working on it

**Comment by timkelty** [comment, marked as answer]
We've adjusted the messaging, included a link to the KB, and most importantly: included the right button :)
<img width="563" alt="CleanShot 2023-06-26 at 16 20 38@2x" src="https://github.com/craftcms/.github/assets/18329/10c20f87-2a4d-4196-ac65-6162afee8a5b">
